src/api/useToken.ts

```diff
@@ -1,17 +1,37 @@
-import { useMutation } from '@tanstack/react-query';
-import { client } from './client';
-import { GetAccessTokenByPassword } from 'generated/auth/data-contracts';
+import { useMutation } from "@tanstack/react-query";
+import { client } from "./client";
+import {
+  GetAccessTokenByPassword,
+  OfflineAccessScope,
+  PasswordGrantType,
+} from "generated/auth/data-contracts";
 
 export const useToken = () => {
   const auth = useMutation({
-    mutationKey: ['getToken'],
-    mutationFn: async (data: GetAccessTokenByPassword) =>
-      await client.auth.getAccessTokens(data, {
-        headers: {
-          'Content-Type': 'application/x-www-form-urlencoded',
-          Accept: 'application/json;charset=UTF-8',
+    mutationKey: ["getToken"],
+    mutationFn: async (
+      data: Pick<GetAccessTokenByPassword, "username" | "password">,
+    ) =>
+      await client.auth.getAccessTokens(
+        {
+          ...data,
+          client_id: "1ec1f3f4-411b-4dc3-ad1c-68196af7e90c",
```

Comment:
this value should be parameterized because it will differ among environments.

Reply:
I've moved it in the env file as `VITE_AUTH_CLIENT_ID`